.github/workflows/linux.yml

```diff
@@ -28,6 +28,8 @@ jobs:
           # Its architecture is different, though, and covered by the Docker job.
           - debian:11
           - debian:12
+          - --platform linux/386 debian:11
+          - --platform linux/386 debian:12
```

Comment:
Rather than this, how about using a fine-grained [`GitHub Matrix`](https://docs.github.com/en/actions/writing-workflows/choosing-what-your-workflow-does/running-variations-of-jobs-in-a-workflow#example-adding-configurations) configuration? The matrix could then have two variables, `distro` and `platform`, with platform set to `amd64` by default, but for Debian you can additionally define this new 32-bit system via `include`.